contributing.md

```diff
@@ -5,34 +5,39 @@ title: Contributing to Bazel
 
 # Contributing to Bazel
 
+There are many ways to help the Bazel project and ecosystem.
 
-## How can I help?
+## Provide feedback
 
-There are many ways to help the Bazel project and ecosystem.
+As you use Bazel, you may find things that can be improved. You can help by:
 
-1. Provide feedback as you use Bazel. As you use it, you may find things
-   that can be improved. Please report them:
-   - Report bugs. In particular, it's important to report any crash or
+   - Reporting bugs. In particular, it's important to report any crash or
```

Comment:
Should we add a link to where folks can report bugs?

Reply:
We could.   http://github.com/bazelbuild/bazel/issues.

Reply:
Done.